mkdocs.yml

```diff
@@ -16,6 +16,7 @@ pages:
         - 'FAQ': 'user-guide/faq.md'
         - 'Troubleshooting': 'user-guide/troubleshooting.md'
         - 'Links': 'user-guide/links.md'
+        - 'Glossary': 'user-guide/glossary.md'
```

Comment:
The glossary is empty at that moment, added it to not forget that we need to have it.